composer.json

```diff
@@ -19,9 +19,9 @@
 
     "require": {
         "php":                      ">=5.3.2",
-        "symfony/framework-bundle": "~2.3|~3.0",
-        "symfony/finder":           "~2.3|~3.0",
-        "symfony/validator":        "~2.3|~3.0",
+        "symfony/framework-bundle": "~2.3|~3.1",
```

Comment:
Poor users of Symfony 3.0 won't be able to install snappy. Why Do we want it? :)